controllers/component_build_controller_pac.go

```diff
@@ -292,7 +293,10 @@ func (r *ComponentBuildReconciler) TriggerPaCBuild(ctx context.Context, componen
 	triggerURL := fmt.Sprintf("%s/incoming?secret=%s&repository=%s&branch=%s&pipelinerun=%s", webhookTargetUrl, secretValue, repository.Name, targetBranch, pipelineRunName)
 	HttpClient := GetHttpClientFunction()
 
-	resp, err := HttpClient.Post(triggerURL, "application/json", nil)
+	// we have to supply source_url as additional param, because PaC isn't able to resolve it for trigger
```

Comment:
Is there a bug for PaC on this? AFAIK it gets resolved for push/pull events.

Reply:
will change the comment

Reply:
mmmm I am not sure if that is bug SRVKP-5837
or it is how it should work

they basically can't identify source_url without explicitly getting it